pkg/agent/sysadvisor/plugin/qosaware/server/cpu/cpu_server.go

```diff
@@ -288,6 +289,16 @@ func (cs *cpuServer) startToGetCheckpointFromCPUPlugin() error {
 }
 
 func (cs *cpuServer) serve() error {
+	cpuAdvisorSocketDir := path.Dir(cs.cpuAdvisorSocketPath)
+
+	err := general.EnsureDirectory(cpuAdvisorSocketDir)
```

Comment:
we may need to emit metrics and add health check for serve functions in the future, just notify, no need to do it now